source/extensions/listener_managers/listener_manager/lds_api.cc

```diff
@@ -46,6 +47,7 @@ void LdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& a
   if (cm_.adsMux()) {
     const std::vector<std::string> paused_xds_types{
         Config::getTypeUrl<envoy::config::route::v3::RouteConfiguration>(),
+        Config::getTypeUrl<envoy::config::route::v3::ScopedRouteConfiguration>(),
```

Comment:
This is not related to the issue. But I think we should pause SRDS when LDS is updated.

Reply:
It's best to split small changes like this in case there are unexpected side-effects.